test/e2e-v2/cases/pulsar/docker-compose.yml

```diff
@@ -22,8 +22,16 @@ services:
       service: oap
     ports:
       - "12800:12800"
+    environment:
+        SW_ENABLE_UPDATE_UI_TEMPLATE: true
+        SW_STORAGE: mysql
+        SW_JDBC_URL: jdbc:mysql://mysql:3306/swtest?allowMultiQueries=true
```

Comment:
May I ask, why suddenly test in the MySQL storage?

Reply:
It is used to debug. I push it to GitHub to share with @liuhaoyang.